pynumaflow/sink/server.py

```diff
@@ -55,12 +55,12 @@ def SinkFn(
         Applies a sink function to a list of datum elements.
         The pascal case function name comes from the generated udsink_pb2_grpc.py file.
         """
-
+        # if there is an exception, we will mark all the responses as a failure
         try:
             rspns = self.__sink_handler(request.elements)
         except Exception as err:
```

Comment:
any reason you didn't try try-catch-else?

Reply:
> @vigith this part is good, actually. We are creating a new list `responses` from `rspns` regardless of the exception or not.

Added the reason in the previous comment! :)